### ISSUE TITLE
Quest 2 on PC

### ISSUE THREAD

**Issue by Albertopina2** [LibreVR/Revive#1693]
Hi, first, thanks a lot for Revive, are a awesome work.

Second, the VRPC users have a great problem, Quest 2  have incredible sales, the future AAA games Will be exclusive on Quest 2, Residente Evil 4, Assassin's Creed, Splinter Cell, etc.

Maybe, with an android virtual machine, emulate Quest 2 on high end PC?

Is time to Revive 2.

Thanks a lot.

**Comment by ahmanwoods** [comment]
I don't think this is possible. Sadly us PCVR users will be missing out this time around.

**Comment by Albertopina2** [comment]
There are several android emulators with which you can play games on your PC, it would be to download the apk, install, map buttons ... I think it can be. Not easy, but possible.

**Comment by Wild-Dog** [comment]
Could this help? https://github.com/MozillaReality/WebXR-emulator-extension

**Comment by optimumbox** [comment]
Just wanted to add an updated question. Since Windows 11 will support sideloading and running APK's could this actually be something viable in the future?
